src/screens/volunteer/VolunteerRegistrationScreen.tsx

```diff
@@ -79,13 +110,12 @@ export const VolunteerRegistrationScreen = ({ navigation }: StackScreenProps<any
                 textContentType="username"
                 autoCapitalize="none"
                 validate
-                rules={{ required: true }}
-                errorMessage={
-                  errors.username && `${texts.volunteer.username} muss ausgefüllt werden`
-                }
+                rules={{ required: texts.volunteer.usernameError }}
```

Comment:
I think the {minLenth} rule needs to be set here. There is an example of this on line 182.

Reply:
true!

> Username should contain at least 4 characters.